### PR TITLE
fix typo in example

### PR DESCRIPTION
**Pull request by ghedamat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ export default Component.extend({
 
 ```js
 // For crappy reasons non inDOM use-cases
-import inDOMObserver from 'ember-runloop-utils/in-dom-observer'
```
Comment (stefanpenner) [inline README.md:38]: the from is also wrong actually

**Comment by ghedamat** [comment]
right, I didn't even check the module path :/ 

I edited from github so maybe you can squashmerge?

otherwise I'll check out the repo and do it there

File: README.md (hunk 1 above)
Comment (ghedamat) [inline README.md:38]: fixed, sry about that

**Comment by stefanpenner** [comment]
👍 thank you sir

**Pull request merged**